imports/ui/body.js

```diff
@@ -0,0 +1,48 @@
+import { Meteor } from 'meteor/meteor';
```

Comment:
These files are just copied from the tutorial right? But I believe you couldn't get them to display?